pystarport/expansion.py

```diff
@@ -0,0 +1,58 @@
+import re
+from typing import Any, Mapping, Optional, Text
+
+from dotenv.variables import parse_variables
+
+
+def expand_posix_vars(obj: Any, variables: Mapping[Text, Optional[Any]]) -> Any:
+    """expand_posix_vars recursively expands POSIX values in an object.
+
+    Args:
+        obj (any): object in which to interpolate variables.
+        variables (dict): dictionary that maps variable names to their value
+    """
+    if isinstance(obj, (dict,)):
+        for key, val in obj.items():
+            obj[key] = expand_posix_vars(val, variables)
+    elif isinstance(obj, (list,)):
+        for index in range(len(obj)):
+            obj[index] = expand_posix_vars(obj[index], variables)
+    elif isinstance(obj, (str,)):
+        obj = _str_to_python_value(_expand(obj, variables))
```

Comment:
why `_str_to_python_value`?

Reply:
Just fix some string to value.

Reply:
This is part of https://github.com/eth-brownie/brownie/blob/master/brownie/_expansion.py

Reply:
Before it fix the int as well, I comment out it. Since I found the expanded yaml is different than the original one, I am not sure it break the config or not (need tests).

Reply:
why convert str to value automatically?

Reply:
Deleted.